Ginger/Ginger/BusinessFlowPages_New/AddActionMenu/SharedRepositoryNavPage.xaml

```diff
@@ -33,6 +33,25 @@
                 </Style>
             </TabControl.Resources>
 
+            <TabItem x:Name="tbActivitiesGroups" Margin="0,0,0,0">
```

Comment:
1- not sure why you created all shared repo tabs again and didn't re-used existing SR page we have (inside Frame), but you already done it so keep it
2- you missing 'x' at begining of x:NAme="" at many controls like "x:Name="tbiActivities"" should be x:Name="xSharedActivitiesTab"

Reply:
Done !